test/spec/client-side-encryption/tests/legacy/fle2-InsertFind-Unindexed.yml

```diff
@@ -80,4 +80,4 @@ tests:
         arguments:
           filter: { encryptedUnindexed: "value123" }
         result:
-          errorContains: "Cannot query"
+          errorContains: "encrypt"
```

Comment:
the yml file has comments for this, we should pull them in just to keep the line numbers in sync